.travis.yml

```diff
@@ -22,12 +22,13 @@ matrix:
         - shellcheck --version
         - shellcheck ci/*.sh
       stage: tools-and-build-and-tier1
-    - name: "Style"
-      install: rustup component add rustfmt-preview
-      script:
-        - rustc ci/style.rs && ./style src
-        - cargo fmt --all -- --check
-      stage: tools-and-build-and-tier1
+    # Disabled due to rust-lang/rustfmt#3341
+    #- name: "Style"
+    #  install: rustup component add rustfmt-preview
+    #  script:
+    #    - rustc ci/style.rs && ./style src
+    #    - cargo fmt --all -- --check
```

Comment:
Would it be possible to disable only this line? We'd like to keep using the other style checker, I forgot about it :D

Reply:
Just the cargo fmt line?

Reply:
Yes